packages/frontend-shared/src/components/StandardModalHeader.vue

```diff
@@ -2,8 +2,11 @@
   <div
     class="sticky top-0 flex items-center justify-between bg-white rounded-t border-b-1px border-b-gray-100 min-h-56px px-24px z-1"
   >
-    <DialogTitle class="text-gray-900 text-18px">
-      <slot /> <span
+    <div>
```

Comment:
Adding an additional container here to remove the 'Need help?' link from the dialog's label while keeping the desired styling. Prior to this change, the dialog would read as "Choose a spec Need help?"; now it's just "Choose a spec".